CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 ## Changelog
 
+0.17.0 (unreleased)
+-------------------
+
+* Update the supported version of Python to 3.12, Pyramid to 2.0.2.
```

Comment:
Question about Pyramid support - nothing changed in the requirements yet, did it?

Reply:
No, but I did update the dev/CI environment to test against Pyramid 2.0.2 so we can claim we support it.

Reply:
I understand now. I guess if I get the actual support in before we release,. then that remains true. 😉

Reply:
Wait, are you saying that currently pyramid_openapi3 does not work with Pyramid 2.0? As the tests seems to say otherwise?

Reply:
No, it totally works - my comment was more about dropping support for 1.x and updating the test suites, sorry for the confusion.